types/classnames/index.d.ts

```diff
@@ -1,22 +1,22 @@
-// Type definitions for classnames
+// Type definitions for classnames 2.2
 // Project: https://github.com/JedWatson/classnames
-// Definitions by: Dave Keen <http://www.keendevelopment.ch>, Adi Dahiya <https://github.com/adidahiya>, Jason Killian <https://github.com/JKillian>
+// Definitions by: Dave Keen <http://www.keendevelopment.ch>
+//                 Adi Dahiya <https://github.com/adidahiya>
+//                 Jason Killian <https://github.com/JKillian>
+//                 Sean Kelley <https://github.com/seansfkelley/>
+//                 Michal Adamczyk <https://github.com/mradamczyk>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare type ClassValue = string | number | ClassDictionary | ClassArray | undefined | null | false;
+type ClassValue = string | number | ClassDictionary | ClassArray | undefined | null | false;
 
 interface ClassDictionary {
 	[id: string]: boolean | undefined | null;
 }
 
-interface ClassArray extends Array<ClassValue> { }
+interface ClassArray extends Array<ClassValue> { } // tslint:disable-line no-empty-interface
```

Comment:
yeah... I vaguely recall hitting that issue initially... might be nice to leave a comment explaining why this is necessary, since it does look awkward